apps/console/src/features/users/pages/users.tsx

```diff
@@ -139,7 +139,7 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
     const [ isListUpdated, setListUpdated ] = useState(false);
     const [ userListMetaContent, setUserListMetaContent ] = useState(undefined);
     const [ userStoreOptions, setUserStoresList ] = useState([]);
-    const [ userStore, setUserStore ] = useState<string>(null);
+    const [ userStore, setUserStore ] = useState<string>("primary");
```

Comment:
Let's use the constant value instead of directly using the string. Check in other places as well

Reply:
Fixed with 1b843f7